Extension/readme.developer.md

```diff
@@ -0,0 +1,358 @@
+that you have added to the `test/scenarios` folder. It will add scenarios that have `assets` and `tests` folders
+(and have been compiled at least once). It does not overwrite or update existing entries.
+
+This saves you the effort of having to manually update the launch.json file.
+
+
+---
+### `yarn code`
+> #### `yarn code <folder|workspace|scenario>` - run VSCode
+
+This runs the isolated VSCode environment, with the cpptools extension that is built in this repo
+
+You can treat this essentially like using `code` from the command line. Settings can be configured, and 
+extensions can be installed into the isolated environment, and will be persisted across runs.
+
+Use `yarn code reset` to remove the isolated environment and start fresh.
+
+> `yarn code reset` - remove the isolated VSCode environment
+
+This will completely remove the isolated VSCode environement for this repository, 
+including cache, extensions, and configuration for the isolated environment.
+
+> `yarn code install` - install the isolated VSCode environment
+
+This installs the isolated VSCode environment if it is not currently installed for this 
+repository. This is done automatically when running the `yarn code`, but can be run manually.
+
+---
+### `yarn generate-native-strings`
+> #### `yarn generate-native-strings` - generates the native strings 
+
+This used to generate nativeStrings.ts and localized_string_ids.h from ./src/nativeStrings.json
+If adding localized strings to the native side, start by adding it to nativeStrings.json and use this to generate the others.
+
+> #### Note
+> The use of the `.scripts/common.ts:write()` function ensures that if the contents don't change the file won't be touched
+
+---
+### `yarn generate-options-schema`
+> #### `yarn generate-options-schema` - generates the options schema
+
+Inserts the options schema into `package.json` from the `tools/OptionsSchema.json` and the `tools/VSSymbolSettings.json` file.
+
+> #### Note
+> The use of the `.scripts/common.ts:write()` function ensures that if the contents don't change the file won't be touched
+
+---
+### `yarn copy-walkthrough-media`
+> #### `yarn copy-walkthrough-media` - copies the walkthrough media
+
+This copies the walkthru media into the `dist` folder.
+
+> #### Note
+> The use of the `.scripts/common.ts:updateFiles()` function ensures that if the contents don't change the files won't be touched
+
+> #### `yarn copy-walkthrough-media watch` - watches for changes and automatically copies the walkthrough media
+
+This will watch for changes to the walkthru files and automatically copy them across when they change.
+
+---
+### `yarn prep`
+> #### `yarn prep` - extension preperation
+
+This will call `yarn copy-walkthrough-media`, `yarn generate-native-strings`, `yarn translations-generate` to ensure
+that the generated files are up to date.
+
+---
+### `yarn import-edge-strings`
+> #### `yarn import-edge-strings` - TBD
```

Comment:
Did you want to add documentation here instead of TBD? This command is only for internal Microsoft developers with access to the VS Localize folder. After the Localization for IntelliSense is updated (see internal docs), run under the Extension folder command `node ./import_edge_strings.js <PathTo-Localize> <PathTo-vscode-cpptools>` to update the `Extension/bin/messages/<languageId>/messages.json` files. For example, https://github.com/microsoft/vscode-cpptools/pull/11193/files. This should be done whenever an IntelliSense update could add more error strings. Otherwise, the localized strings will be missing.

Reply:
I've removed those, since they don't need to be in the public doc.